src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -6457,10 +6457,10 @@ End Class
             Assert.Equal("(Item1 As System.Int32, Item2 As System.String, Item3 As System.Int32, Item4 As System.String, Item5 As System.Int32, Item6 As System.String, Item7 As System.Int32, Item8 As System.String, Item9 As System.Int32)",
                          tuple9WithNames.ToTestDisplayString())
 
-            Assert.Equal(New String() {"Item1", "Item2", "Item3", "Item4", "Item5", "Item6", "Item7", "Item8", "Item9"}, tuple9WithNames.TupleElementNames)
+            Assert.Equal(New String() {"Item1", "Item2", "Item3", "Item4", "Item5", "Item6", "Item7", "Item8", "Item9"}, GetTupleElementNames(tuple9WithNames))
 
             Assert.Equal(New String() {"System.Int32", "System.String", "System.Int32", "System.String", "System.Int32", "System.String", "System.Int32", "System.String", "System.Int32"},
-                         tuple9WithNames.TupleElementTypes.Select(Function(t) t.ToTestDisplayString()))
+                         tuple9WithNames.TupleElements.Select(Function(t) t.Type.ToTestDisplayString()))
```

Comment:
Please add a helper method for `tuple.TupleElements.Select(Function(t) t.Type.ToTestDisplayString())`.